build hunspell-sys also on Windows (#2)
euclio/hunspell-sys: README.md

Rust bindings for the [hunspell] C API.

[hunspell]: https://hunspell.github.io/

## Building

> By default `hunspell-sys` searches while building for hunspell library with `pkg-config`.

Reviewer: ```suggestion
By default `hunspell-sys` searches for a hunspell library installation with `pkg-config`.
```
